### PR TITLE
feat: add MIN_L1_TX_SIZE configuration

### PR DESCRIPTION
**Pull request by cfromknecht**
**Description**
Adds a `MIN_L1_TX_SIZE` configuration for controlling the minimum tx batch size.

**Metadata**
- Fixes ENG-1948

### CODE REVIEW

File: ops/envs/batch-submitter.env [hunk 1]
```diff
@@ -4,6 +4,7 @@ ETH_NETWORK_NAME=clique
 LOG_LEVEL=debug
 BATCH_SUBMITTER_LOG_LEVEL=debug
 BATCH_SUBMITTER_LOG_TERMINAL=true
+BATCH_SUBMITTER_MIN_L1_TX_SIZE=32
```
Comment (smartcontracts) [inline ops/envs/batch-submitter.env:7]: Is this in bytes?

Comment (tynes) [reply ops/envs/batch-submitter.env:7]: yes it is

Comment (tynes) [reply ops/envs/batch-submitter.env:7]: and it also includes the function selector

**Comment by tynes** [comment]
Do you mind adding a logline that logs the min and max tx sizes at startup?

**Review by tynes — APPROVED** [review]

File: ops/envs/batch-submitter.env (hunk 1 above)
Comment (optimisticben) [reply ops/envs/batch-submitter.env:7]: Maybe include the unit with the config option, like metrics?
`BATCH_SUBMITTER_MIN_L1_TX_BYTES`
`BATCH_SUBMITTER_POLL_INTERVAL_MS`

**Review by mslipper — APPROVED** [review]

**Pull request merged**